src/init.cpp

```diff
@@ -304,18 +306,13 @@ void PrepareShutdown(NodeContext& node)
 
     // After all scheduled tasks have been flushed, destroy pointers
     // and reset all to nullptr.
-    node.netfulfilledman = nullptr;
-    ::netfulfilledman.reset();
     node.mn_metaman = nullptr;
     ::mmetaman.reset();
-    node.dstxman = nullptr;
-    ::dstxManager.reset();
     node.mn_sync = nullptr;
     ::masternodeSync.reset();
-    node.sporkman = nullptr;
-    ::sporkManager.reset();
-    node.govman = nullptr;
-    ::governance.reset();
+    node.sporkman.reset();
+    node.govman.reset();
+    node.netfulfilledman = nullptr;
```

Comment:
shouldn't this be
```suggestion
    node.netfulfilledman.reset();
```
?

Reply:
Resolved in latest push